src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,72 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.services.DriverService;
+import mate.jdbc.services.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        testManufacturers();
+        System.out.println();
+        testDrivers();
+    }
+
+    private static void testManufacturers() {
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        System.out.println("All manufacturers:");
+        List<Manufacturer> manufacturers = manufacturerService.getAll();
+        manufacturers.stream()
+                .map(Manufacturer::toString)
+                .forEach(System.out::println);
```

Comment:
redundant code. i think it could be just manufacturers.forEach(System.out::println)